src/pfe/file-watcher/server/test/unit-test/tests/projectExtensions.module.test.ts

```diff
@@ -358,6 +364,12 @@ export function projectExtensionsTestModule(): void {
             const expectedResultProjectCapabilities = combinations[combo]["resultProjectCapabilities"];
 
             it(combo + " => getProjectHandler & getProjectCapabilities: projectID " + projectID, async () => {
+                if (combo == "combo6") {
+                    expect(fs.statSync(extensionsPath)).to.exist;
```

Comment:
The existence of the required files/dirs has already checked  in before block. No need to recheck